model_definitions/dfd4052e-f91b-4aa5-9c79-f26d649dd931/model_modules/training.py

```diff
@@ -25,6 +25,7 @@ def train(data_conf, model_conf, **kwargs):
                    password=os.environ["AOA_CONN_PASSWORD"],
                    database=data_conf["schema"] if "schema" in data_conf and data_conf["schema"] != "" else None)
 
+    check_sto_version()
     cleanup_cli(model_version)
```

Comment:
maybe add the table name argument so its clear. In fact we can probably have the table name defined here at the top as we use it later in the code when we save the models also?

Reply:
Sure